web3/_utils/method_formatters.py

```diff
@@ -458,10 +453,13 @@ def method_filter(formatter_dict, method):
 
 @to_tuple
 def get_request_formatters(method):
```

Comment:
@pipermerriam before I do the result/error formatters, is this along the lines of what you were thinking?

Reply:
Not quite.  This still results in the formatters being a tuple of formatter functions instead of what I think would be a better API of it being a single formatter function.

```python
def get_request_formatter(method):
    formatters = ... # use a rough equivalent of the current logic to grab the right formatter functions from the PYTHONIC and ABI formatters
    return compose(*reversed(formatters))  # I think the `reversed` is needed to have them apply in the right order.
```

This results in a single formatter function for each method.

Reply:
Aha! Okay, that makes sense to me. I like that.